### PR TITLE
some typos and minor refactor

### PR DESCRIPTION
**Pull request by aldenml**

### CODE REVIEW

**Comment by aldenml** [comment]
Hi @arvidn, if I'm reading correctly, the errors in the CI are not due to these changes. Is that right?

File: include/libtorrent/torrent_handle.hpp [hunk 1]
```diff
@@ -293,8 +293,8 @@ namespace aux {
 		// omitted then it is implicitly set to the end of the file the first
 		// piece is set to. If the torrent metadata has not been downloaded yet
 		// then the functions do nothing.
-		void set_sequential_range(piece_index_t const first_piece, piece_index_t const last_piece) const;
-		void set_sequential_range(piece_index_t const first_piece) const;
+		void set_sequential_range(piece_index_t first_piece, piece_index_t last_piece) const;
+		void set_sequential_range(piece_index_t first_piece) const;
```
Comment (sledgehammer999) [inline include/libtorrent/torrent_handle.hpp:297]: Won't this break the API/ABI?

Comment (aldenml) [reply include/libtorrent/torrent_handle.hpp:297]: the compiler ignores this particular use of `const` (to an `int`) in the declaration and the lint suggests removing it. It's still present in the definition.

**Review by arvidn — APPROVED** [review]

File: include/libtorrent/torrent_handle.hpp (hunk 1 above)
Comment (arvidn) [reply include/libtorrent/torrent_handle.hpp:297]: right. top level `const` is redundant in function declarations.

**Pull request merged**